data/resources/ui/login.ui

```diff
@@ -240,6 +240,28 @@
                                 </style>
                               </object>
                             </child>
+                            <child>
+                              <object class="GtkListBox">
+                                <child>
+                                  <object class="AdwActionRow" id="code_resend_action_row">
```

Comment:
Instead of adding an action row + button I would prefer just a simple button with .pill class. To display the information like waiting X seconds I'd use the same button but with sensitive=false and .flat class.

Reply:
> To display the information like waiting X seconds I'd use the same button but with sensitive=false and .flat class.

Or, even better, a separate GtkLabel with .dim-label, in that case we would need a GtkStack to display either the button or the label.